packages/@guardian/source-react-components-development-kitchen/package.json

```diff
@@ -33,6 +27,7 @@
   "devDependencies": {
     "@guardian/libs": "^3.2.1",
     "mini-svg-data-uri": "^1.3.3",
+    "np": "^7.5.0",
```

Comment:
Assuming that we're going to do something similar with the other packages in time, would it be better to move this dependency to the root `package.json` or shall we do that if/when we add it to the next package?

Reply:
i think because `np` _is_ a dev dep of this specific package we should include it here. it may be that we end up with lots of versions across the packages, but this means we dont have to worry about the version that's installed being in sync, deleting it if we move other packages to a different process etc.

Reply:
Fair enough. 👍